.github/workflows/pr_shelter.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         mkdir -p /root/inclavare-containers/${{ matrix.tag }};
         cd inclavare-containers-$RUNE_VERSION/enclave-tls;
         make SGX=1 && make install;
-        cp -rf /opt/enclave-tls /root/inclavare-containers/${{ matrix.tag }}'
+        cp -rf /usr/share/enclave-tls /root/inclavare-containers/${{ matrix.tag }}'
```

Comment:
We install bin into `/usr/share/enclave-tls/samples/`. The libraries and head files are still installed in `/opt/enclave-tls`.  So only copy `/usr/share/enclave-tls` is not enough.

Reply:
For this directory, i only need enclave-tls-server.bin and stub_enclave_signed.so for test. after make and make install.  Other components are still under /opt/enclave-tls.

Reply:
Separate the two commit into two PRs.